content/post/deux-sprints-une-version.en.md

```diff
@@ -0,0 +1,41 @@
+
+<!--more-->
+
+## Link to contributing person-authorities
+
+A new field `contribution` replaces the `author` field when cataloguing documents, in order to comply with current standards, in particular RDA cataloging rules. Documents are put in relation with contributing agents, whose authorities come from verified repositories (IdRef and GND). The role of each agent in the production of a document is also indicated by an RDA relationship code. If the authority is not found in the repositories, it can also be recorded manually as a character string in the same field.
+
+{{< figure src="/img/contribution.jpg" caption="The `contribution` field newly added to the document data model">}}
+
+## Circulation density under control
+
+A great deal of behind-the-scenes work has been done on the backend of the circulation module, in order to solve the problems observed during the first pilot tests in March 2020 and to correctly implement all the [circulation actions](https://github.com/rero/rero-ils/blob/dev/doc/circulation/actions.md) identified, on the one hand. On the other hand, automated tests have also been developed in order to be able to check the proper functioning of this module with each new version and to identify any unexpected errors when work is carried out on other parts of the software.
+
+## Need help ?
+
+With the ongoing progress of RERO ILS' functionalities, it was necessary to document its operation for the end-user. This is now done with a [help platform](https://ils.test.rero.ch/help/home/) accessible from the professional and public interfaces of RERO ILS. This application offers, on the same platform, the public and professional help of RERO ILS to guide users and support their training on the system.
+
+{{< figure src="/img/public-help.JPG" caption="The RERO ILS help interface developed with [flask-wiki](https://github.com/rero/flask-wiki)">}}
+
+## Listing of new acquisitions
+
+A field that should be interesting for the highlighting of collections has been implemented in the item data model: `new_acquisition`. This field is used to indicate whether an item should be part of _new acquisitions_ according to each library's criteria. This allows to exclude certain documents from this list (heritage documents, repurchases, older books, or others). This field also includes a date that can be used to build URLs to display the list of a library's new acquisitions. This list can be filtered by date, but also by other criteria (location, status, etc.), which offers great flexibility.
+
+A common use case of this feature is to be able to direct to the list of new acquisitions for a given month directly from a library's website. The URLs created are permanent and point directly to a list of documents in the public interface of RERO ILS. The construction of URLs for these lists is also documented in the [professional help](https://ils.test.rero.ch/help/recherche/#nouvelles-acquisitions).
```

Comment:
Suggestion: remplacer:
~~A common use case of this feature is to be able to direct to the list of new acquisitions for a given month directly from a library’s website.~~
par
A common use case of this feature is to be able to include in the library’s website a link to the list of new acquisitions for a given month.